service/controller/v15/encrypter/vault/vault.go

```diff
@@ -319,68 +395,6 @@ func (e *Encrypter) Decrypt(key, ciphertext string) (string, error) {
 	return string(plaintext), nil
 }
 
```

Comment:
Methods below replaced with `Ensure` ones. Logic is similar. They are just idempotent.